### DISCUSSION TITLE
Is it possible to implement this on an LXC Container without using docker?

### DISCUSSION THREAD

**Discussion by FleketBreaker** [wg-easy/wg-easy#1343] [Q&A] [answered]
Hi! I'm really interested in using wg-easy, but I would like to implement it in an LXC container without using Docker. Mainly because I want to avoid having a container within a container (LXC -> Docker).

If anyone can give me a hint on where to start looking for this and whether it's even possible, I would really appreciate it.

**Comment by eandersons** [comment, marked as answer]
There are a couple of places you could look at for an inspiration:

- the [Dockerfile](/wg-easy/wg-easy/blob/master/Dockerfile) provides information on how image is being prepared and which command is executed when starting a container;
- the wiki page [Using WireGuard Easy without Docker](/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker).
